actions/project.go

```diff
@@ -70,6 +71,36 @@ func DownloadTemplate(c *cli.Context) {
 	}
 }
 
+// testIsExtension tests if a project is an extension project and run associated commands as necessary
+func testIsExtension(projectPath string) (string, error) {
```

Comment:
Could this function be called something else? Only suggesting this as when we unit test the functions begin `TestFooBar()` and this func name could be miss read / miss leading as it is not actually a test